### DISCUSSION TITLE
Suggestion Thread

### DISCUSSION THREAD

**Discussion by xylabolos** [xylabolos/17F-Madis#8] [Ideas]
Comment your suggestion here on how we're going to present our stopwatch!

**Comment by escuadrafrancis** [comment]
Keep it simple

**Reply by rofelmaetayoto** [reply]
agree

**Reply by xylabolos** [reply]
What kind of concept do you think is the best for our project?

**Reply by abriellecania** [reply]
wala ako idea huhu
